AI/BattleAI/BattleAI.cpp

```diff
@@ -179,94 +125,9 @@ BattleAction CBattleAI::activeStack( const CStack * stack )
 	return BattleAction::makeDefend(stack);
 }
 
-void CBattleAI::actionFinished(const BattleAction &action)
-{
-	print("actionFinished called");
-}
-
-void CBattleAI::actionStarted(const BattleAction &action)
-{
-	print("actionStarted called");
-}
-
-void CBattleAI::battleAttack(const BattleAttack *ba)
-{
-	print("battleAttack called");
-}
-
-void CBattleAI::battleStacksAttacked(const std::vector<BattleStackAttacked> & bsa)
-{
-	print("battleStacksAttacked called");
-}
-
-void CBattleAI::battleEnd(const BattleResult *br)
-{
-	print("battleEnd called");
-}
-
-void CBattleAI::battleNewRoundFirst(int round)
-{
-	print("battleNewRoundFirst called");
-}
-
-void CBattleAI::battleNewRound(int round)
-{
-	print("battleNewRound called");
-}
-
-void CBattleAI::battleStackMoved(const CStack * stack, std::vector<BattleHex> dest, int distance)
-{
-	print("battleStackMoved called");;
-}
-
-void CBattleAI::battleSpellCast(const BattleSpellCast *sc)
-{
-	print("battleSpellCast called");
-}
-
-void CBattleAI::battleStacksEffectsSet(const SetStackEffect & sse)
-{
-	print("battleStacksEffectsSet called");
-}
-
-void CBattleAI::battleStart(const CCreatureSet *army1, const CCreatureSet *army2, int3 tile, const CGHeroInstance *hero1, const CGHeroInstance *hero2, bool Side)
-{
-	print("battleStart called");
-	side = Side;

```

Comment:
Yeah it's looks like this one is useful. With new code CGameHandler::makeCustomAction complain about missing side in some cases.